launchable/test_runners/robot.py

```diff
@@ -49,6 +48,19 @@ def parse_func(p: str) -> ET.ElementTree:
                 if status == "NOT_RUN" or nested_status == 'NOT_RUN':
                     skipped = ET.SubElement(testcase, "skipped")  # noqa: F841
 
+    original_tree = ET.parse(p)
+    testsuite = ET.Element("testsuite", {"name": "robot"})
+
+    SUITE_TAG_NAME = "suite"
+    for suites in original_tree.findall(SUITE_TAG_NAME):
+        nested_suites = suites.findall(SUITE_TAG_NAME)
+
+        if nested_suites:
+            for suite in nested_suites:
+                parse_suite(suite)
+        else:
+            parse_suite(suites)
```

Comment:
How about adding comments as follows?
```suggestion
        if nested_suites:
            for suite in nested_suites:
                parse_suite(suite)
        else:
            # for a single-file test run
            parse_suite(suites)
```